bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/RifLayoutsProcessor.java

```diff
@@ -948,6 +994,39 @@ private static TypeName selectJavaPropertyType(RifField rifField) {
 		return annotations;
 	}
 
+	/**
+	 * @param List<String>
+	 *            the {@link RifField} to create an additional Annotated database
+	 *            field for
+	 * @return an ordered {@link List} of {@link RifField}s representing the
+	 *         additional fields that need to be stored to the database via JPA
+	 * 
+	 * @throws MalformedURLException
+	 */
+	private static List<RifField> createDetailsForAdditionalDatabaseFields(List<String> additionalDatabaseFields)
+			throws MalformedURLException {
+		List<RifField> addlDatabaseFields = new ArrayList<RifField>();
+
+		for (String additionalDatabaseField : additionalDatabaseFields) {
+			if (additionalDatabaseField.contentEquals("unhashedHicn")) {
+				RifField unhashedHicn = new RifField("unhashed_Hicn", RifColumnType.CHAR, Optional.of(64),
```

Comment:
The field name's case here has me confused a bit. Why's the variable name like that?

Is this creating a not-in-the-spreadsheet field to account for the fact that the unhashed HICN field is synthetic? If so, I'd probably match the casing of the other fields in the spreadsheet, which I'd guess is all-caps-with-underscores?

Note also that the field name and length will likely need to be adjusted to match the changes I suggested over in https://github.com/CMSgov/bluebutton-data-model/pull/45.